src/components/Nav/Nav.css

```diff
@@ -0,0 +1,23 @@
+nav {

```

Comment:
The content is visible behing the navbar making the items in the navbar difficult to read.
![image](https://github.com/iO-Academy/23-sept-sarcasticfringeheads-bookswap-fe/assets/9077806/a217f5cc-5ced-4ce8-af4a-4e60fd965b10)